brave-lists/clean-urls.json

```diff
@@ -145,10 +145,14 @@
     },
     {
         "include": [
-            "*://*.twitter.com/*"
+            "*://*.twitter.com/*",
+            "*://*.x.com/*",
+            "*://x.com/*"
         ],
         "exclude": [
-            "*://*.twitter.com/i/redirect?*"
+            "*://*.twitter.com/i/redirect?*",
+            "*://*.x.com/*",
```

Comment:
This is going to exclude everything in  the `x.com` domain. Instead, it should be:
```
*://*.x.com/i/redirect?*
```